src/test/java/bio/terra/service/filedata/DrsDaoTest.java

```diff
@@ -0,0 +1,65 @@
+package bio.terra.service.filedata;
+
+import static org.hamcrest.MatcherAssert.assertThat;
+import static org.hamcrest.Matchers.equalTo;
+import static org.hamcrest.Matchers.samePropertyValuesAs;
+import static org.junit.Assert.assertNull;
+
+import bio.terra.common.EmbeddedDatabaseTest;
+import bio.terra.common.fixtures.AuthenticationFixtures;
+import bio.terra.common.iam.AuthenticatedUserRequest;
+import bio.terra.service.filedata.DrsDao.DrsAlias;
+import bio.terra.service.filedata.DrsDao.DrsAliasSpec;
+import java.time.Instant;
+import java.util.List;
+import java.util.UUID;
+import org.junit.jupiter.api.Tag;
+import org.junit.jupiter.api.Test;
+import org.junit.runner.RunWith;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.boot.test.autoconfigure.web.servlet.AutoConfigureMockMvc;
+import org.springframework.boot.test.context.SpringBootTest;
+import org.springframework.test.context.ActiveProfiles;
+import org.springframework.test.context.junit4.SpringRunner;
+
+@RunWith(SpringRunner.class)
+@SpringBootTest
+@AutoConfigureMockMvc
+@ActiveProfiles({"google", "unittest"})
+@Tag("bio.terra.common.category.Unit")
+@EmbeddedDatabaseTest
```

Comment:
I think this would be a good one to convert to JUnit 5 as well, if possible!

Here's a similar test which uses test slicing to opt in the needed bean(s):
https://github.com/DataBiosphere/jade-data-repo/blob/develop/src/test/java/bio/terra/app/controller/Oauth2ApiControllerTest.java

Reply:
Ooh good call.  Will do!

Reply:
Hmm...haven't been able to get it to work.  The problem with this one I think is that this one needs the DB which is more complicated than the Oauth one

Reply:
I also think it's already using 5 (certainly using the 5 function format)

Reply:
Ah, you know, I think it was pretty close to JUnit5 already.  If you remove `@RunWith(SpringRunner.class)` which is a JUnit 4 annotation, it should still work.  Then all of your annotations look to be JUnit5.

Totally fair on the test slicing.  I was working on a DAO test and couldn't get it to work with test slicing either, I'm sure there's a way but I'd elected to move on.

Reply:
rock on